pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -2094,7 +2094,7 @@
         heading: "Music Lab at a glance"
         ages: "10+"
         level: "Beginner"
-        make: "Simple Javascript apps"
+        make: "Block-based music mixes"
```

Comment:
just checking -- we're good to update without waiting for translations?

Reply:
I just asked Michael IRL and he said it's ok to update w/o waiting 🙌